sdwan_profile_parcels.tf

```diff
@@ -0,0 +1,11 @@
+resource "sdwan_cli_config_profile_parcel" "cli_config_profile_parcel" {
+  for_each = {
+    for cli in try(local.feature_profiles.cli_profiles, {}) :
+    "${cli.name}-config" => cli
+    if lookup(cli, "config", null) != null
+  }
+  name               = each.value.config.name
```

Comment:
Should we maybe make this name optional and use `"${each.value.name}-config"` by default?

Reply:
done in https://github.com/netascode/terraform-sdwan-nac-sdwan/pull/28/commits/a2066dc5b5b32f7dcade4fb6935a48139de78e4e